<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vigneshs-MacBook-Air-3.local" ignored="2" name="Test(failed)" tests="2" failures="1" timestamp="2020-09-15T16:09:03 IST" time="0.077" errors="0">
  <testcase name="thirdTestCase" time="0.015" classname="SeleniumInterviewQuestions.ReRunFailedTestCases">
    <failure type="javax.management.RuntimeErrorException">
      <![CDATA[javax.management.RuntimeErrorException
at SeleniumInterviewQuestions.ReRunFailedTestCases.thirdTestCase(ReRunFailedTestCases.java:27)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- thirdTestCase -->
  <testcase name="secondTestCase" time="0.002" classname="SeleniumInterviewQuestions.ReRunFailedTestCases"/>
  <testcase name="fourthTestCase" time="0.0" classname="SeleniumInterviewQuestions.ReRunFailedTestCases">
    <ignored/>
  </testcase> <!-- fourthTestCase -->
  <testcase name="firstTestCase" time="0.0" classname="SeleniumInterviewQuestions.ReRunFailedTestCases">
    <ignored/>
  </testcase> <!-- firstTestCase -->
</testsuite> <!-- Test(failed) -->
